org.eclipse.jdt.ui/plugin.properties

```diff
@@ -1219,6 +1219,7 @@ PropertyKeyHyperlinkDetector= Java Property Key
 JavaElementHyperlinkDeclaredTypeDetector= Open Declared Type
 JavaElementHyperlinkReturnTypeDetector= Open Return Type 
 JavaElementHyperlinkSuperImplementationDetector= Open Super Implementation 
+OpenCallHierarchyHyperlinkDetector=Open Call Hierarchy
```

Comment:
Great idea. I've added missing externalisation key in the plugin.properties & pushed.
Once tests will be green, we can merge.